<launch>
        <arg name="veh"/>
        <arg name="local" default="true" doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>
        <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />
        <arg name="camera" default="true" doc="camera set to false means only apriltags no camera nodes are launched"/>
    <arg name="live" default="true" doc="live set to false means we don't run the actual camera (i.e. running from a log file" />
    <arg name="width" default="640" doc="width of compressed image"/>
    <arg name="height" default="480" doc="width of compressed image"/>
    <arg name="tango" default="false" doc="subscribe tango image"/>


    <!--remap from="/camera/color/image_rect_color" to="/zed/rgb/image_rect_color"/--> 
    <!-- Run on local (laptop) -->
        <node if="$(arg local)" ns="$(arg veh)" pkg="text_navigation" type="fcn_node.py" name="fcn_node" output="screen" clear_params="true" required="true">
            <rosparam command="load" file="$(find text_navigation)/config/baseline/text_navigation/cnn_node/default.yaml"/>
        </node>

    <!-- Run on remote (vehicle) -->
    <include unless="$(arg local)" file="$(find duckietown)/machines"/>
        <node unless="$(arg local)" ns="$(arg veh)" machine="$(arg veh)" pkg="text_navigation" type="fcn_node.py" name="fcn_node" output="screen" clear_params="true" required="true">
            <rosparam command="load" file="$(find text_navigation)/config/baseline/text_navigation/cnn_node/default.yaml"/>
        </node>	


</launch>
